docs/fastdds/ros2/discovery_server/ros2_discovery_server.rst

```diff
@@ -318,12 +317,21 @@ Compare Discovery Server with Simple Discovery
 In order to compare the ROS2 execution using *Simple Discovery* or *Discovery Service*, two scripts that
 execute a talker and many listeners and analyze the network traffic during this time are provided.
 For this experiment, ``tshark`` is required to be installed on your system.
+The configuration file is mandatory in order to avoid using intra-process mode.
+
+.. note::
+
+    These scripts require a Discovery Server closure feature that is only available from versions newer than the
+    one used in ROS 2 Foxy.
```

Comment:
I'd specify the actual Fast DDS version here

Reply:
It is specified in the very next line, I think it is enough

Reply:
The thing is that the version in Foxy might change in the future. Maybe it's better to remove the reference to Foxy altogether.